packages/components/menu/Menu.mdx

```diff
@@ -146,6 +144,34 @@ function ControlledMenu() {
 }
 ```
 
+### Controlled Menu with custom trigger onClick callback
+
+In most of the cases you will use the controlled approach described [above](#controlled-menu).
+But if you have to provide your own toggle menu logic on the trigger component, you can do this by NOT providing `onOpen` callback to the Menu component and providing `onClick` callback on trigger component.
```

Comment:
```suggestion
But if you have to provide your own toggle menu logic on the trigger component, you can do this by providing `onClick` callback on the trigger component.
```

Reply:
I'd love to accept your variant, but user should clearly understand that only in case if `onOpen` prop is undefined, `onClick` with toggling logic will work properly. Do you have any suggestions how to explain it better?

Reply:
hmmm, good point. I am not sure tbh. I just prefer when we say what they should do instead of what not to do. But yeah, no worries, I'm fine with you version :)